samples/idea-latest.yaml

```diff
@@ -0,0 +1,36 @@
+kind: DevWorkspace
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  name: idea-latest
+spec:
+  started: false
+  template:
+    projects:
+      - name: web-nodejs-sample
+        git:
+          remotes:
+            origin: "https://github.com/vitaliy-guliy/web-nodejs-sample.git"
+            # origin: "https://github.com/che-samples/web-nodejs-sample.git"
+          checkoutFrom:
+            revision: "devfilev2"
```

Comment:
We should consider a different sample, as this is a javascript project. Opening `app.js` in the editor leads to a warning that "IDEA provides other IDEs for editing javascript files"

Reply:
https://github.com/che-samples/python-hello-world/tree/devfilev2 would be not bad candidate